packages/esm-patient-test-results-app/src/filter/filter-context.tsx

```diff
@@ -22,47 +22,18 @@ const initialContext = {
   updateParent: () => {},
 };
 
-interface StateProps {
-  checkboxes: { [key: string]: boolean };
-  parents: { [key: string]: string[] };
-  roots: { [key: string]: any }[];
-}
-interface FilterContextProps {
-  state: StateProps;
-  checkboxes: { [key: string]: boolean };
-  parents: { [key: string]: string[] };
-  roots: TreeNode[];
-  tests: { [key: string]: any };
-  lowestParents: { display: string; flatName: string }[];
-  timelineData: { [key: string]: any };
-  activeTests: string[];
-  someChecked: boolean;
-  initialize: any;
-  toggleVal: any;
-  updateParent: any;
-}
-
-interface FilterProviderProps {
-  roots: any[];
-  children: React.ReactNode;
-}
-
-interface obsShape {
-  [key: string]: any;
-}
-
 const FilterContext = createContext<FilterContextProps>(initialContext);
 
 const FilterProvider = ({ roots, children }: FilterProviderProps) => {
   const [state, dispatch] = useReducer(reducer, initialState);
 
   const actions = useMemo(
     () => ({
-      initialize: (trees) => dispatch({ type: 'initialize', trees: trees }),
-      toggleVal: (name) => {
+      initialize: (trees: TreeNode[]) => dispatch({ type: 'initialize', trees: trees }),
```

Comment:
Hi @ZacButko! 
We are using `Array<Type>` instead of `Type[]` in the project. I know these are the same things, but for the convention, we should use `Array<>`.
What do you think?